src/components/Layer.js

```diff
@@ -675,7 +685,7 @@ export class Layer extends Component {
 							{this.renderConfigCypher()}
 							{this.renderConfigSpatial()}
 
-							<h4> > Map rendering</h4>
+							<h4>{' > Map rendering'}</h4>
```

Comment:
Just out of curiosity, why this change (and the others of the same kind above)?

Reply:
Cheap syntactical fix suggested by my editor. > in the string as written before errors it clashes with html tags closing >

Reply:
Ok, good reason :)